Loop/Managers/DeviceDataManager.swift

```diff
@@ -1339,69 +1340,70 @@ extension DeviceDataManager {
         case .bolusEntry(let bolusAmount):
-            log.default("Enacting remote bolus entry: %{public}@", String(describing: bolusAmount))
+            log.default("Remote Notification: Enacting bolus entry: %{public}@", String(describing: bolusAmount))
             
             //Remote bolus requires validation from its remote source
             guard remoteDataServicesManager.validatePushNotificationSource(notification) else {
                 NotificationManager.sendRemoteBolusFailureNotification(for: RemoteCommandError.invalidOTP, amount: bolusAmount)
-                log.info("Could not validate notification: %{public}@", String(describing: notification))
+                log.error("Remote Notification: Could not validate notification: %{public}@", String(describing: notification))
                 return
             }
             
             guard let maxBolusAmount = loopManager.settings.maximumBolus else {
                 NotificationManager.sendRemoteBolusFailureNotification(for: RemoteCommandError.missingMaxBolus, amount: bolusAmount)
-                log.default("No max bolus detected. Aborting...")
+                log.error("Remote Notification: No max bolus detected. Aborting...")
                 return
             }
             
             guard bolusAmount.isLessThanOrEqualTo(maxBolusAmount) else {
                 NotificationManager.sendRemoteBolusFailureNotification(for: RemoteCommandError.exceedsMaxBolus, amount: bolusAmount)
-                log.default("Remote bolus higher than maximum. Aborting...")
+                log.error("Remote Notification: Bolus exceeds maximum allowed. Aborting...")
                 return
             }
             
             // For remote boluses, assume manual no recommendation.
             self.enactBolus(units: bolusAmount, activationType: .manualNoRecommendation) { error in
                 if let error = error {
+                    self.log.error("Remote Notification: Error adding bolus: %{public}@", String(describing: error))
                     NotificationManager.sendRemoteBolusFailureNotification(for: error, amount: bolusAmount)
                 } else {
                     NotificationManager.sendRemoteBolusNotification(amount: bolusAmount)
                 }
             }
         case .carbsEntry(let candidateCarbEntry):
-            log.default("Adding carbs entry.")
+            log.default("Remote Notification: Adding carbs entry: %{public}@", String(describing: candidateCarbEntry))
             
             let candidateCarbsInGrams = candidateCarbEntry.quantity.doubleValue(for: .gram())
             
             //Remote carb entry requires validation from its remote source
             guard remoteDataServicesManager.validatePushNotificationSource(notification) else {
                 NotificationManager.sendRemoteCarbEntryFailureNotification(for: RemoteCommandError.invalidOTP, amountInGrams: candidateCarbsInGrams)
```

Comment:
I really want to see how old the expired OTP codes are here. Add a log when this failure happens should let me infer in from user's logs. 

If OTPs are consistently 2-3 minutes old during a failure, we could consider extending the age of OTPs we accept to account for that. I think that would be a good interim step to making our remote capabilities more reliable until a command queue solution is developed.